AmplifyPlugins/Core/AWSPluginsCore/Model/Support/Model+GraphQL.swift

```diff
@@ -169,6 +169,8 @@ extension Model {
         } else if let optionalModel = value as? Model?,
                   let modelValue = optionalModel {
             return modelValue.identifier(schema: modelSchema).values
+        } else if let lazyModelValue = value as? (any LazyModelMarker) {
+            print("FOUND A LAZY MODEL \(lazyModelValue)")
```

Comment:
this is used for APIPlugin's create APIs, need to unit test create mutations GraphQLRequest for `let comment = Comment(post: post)` instances that the create mutation successfully creates a GraphQLRequest. Then test that the integration of that works with AppSync backend.